cfme/tests/control/test_bugs.py

```diff
@@ -147,6 +155,48 @@ def hardware_reconfigured_alert(alert_collection):
     alert.delete()
 
 
+@pytest.fixture
+def setup_disk_usage_alert(alert_collection, alert_profile_collection, appliance):
+    if appliance.version < "5.10":
+        pytest.skip("The fix for BZ 1658670 has not been backported to CFME 5.9")
```

Comment:
Is this fix not going to be backported? If it is not, I would suggest adding `@pytest.mark.ignore_stream()`. If it is going to be backported eventually, you can use the `@pytest.mark.meta(blockers=[BZ(1658670, forced_streams=["5.9"])])`

Reply:
@jawatts, it doesn't look like it will be backported, so I will add `@pytest.mark.ignore_stream()` to the test.